.github/workflows/manylinux_2_28.yml

```diff
@@ -254,6 +285,23 @@ jobs:
       affected-components: ${{ needs.smart_ci.outputs.affected_components }}
       python-version: ${{ matrix.python-version }}
 
+  NodeJS_API_Tests:
+    name: NodeJS API tests
+    needs: [ Docker, Build, Smart_CI ]
+    uses: ./.github/workflows/job_openvino_js.yml
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - nodejs-version: 20
+            image: ${{ fromJSON(needs.docker.outputs.images).ov_test.ubuntu_22_04_x64 }}
+          - nodejs-version: 22
+            image: ${{ fromJSON(needs.docker.outputs.images).ov_test.ubuntu_22_04_x64 }}
+    with:
+      runner: 'aks-linux-4-cores-16gb'
+      image: ${{ matrix.image }}
+      nodejs-version: ${{ matrix.nodejs-version }}
+
   Overall_Status:
     name: ci/gha_overall_status_manylinux_2_28
     needs: [Smart_CI, Build, Python_API_Tests]
```

Comment:
Should NodeJS_API_Tests stage be required for merge? If so, need to add it here (and double-check that it will work as expected for matrix build)

Reply:
Thanks, I updated it